components/collections/src/codepointtrie/cptrie.rs

```diff
@@ -232,7 +233,11 @@ macro_rules! w(
 
```

Comment:
Please include the specific benchmarks that motivate this change.

Ideally in a separate PR so that we can see the change across the PR in the dashboards.

This is important because:

1. It lets the reviewers reproduce the motivating numbers
2. It lets future team members change the code further without undoing the performance gains

(2) is important because, as Rust makes more things safe, maybe in the future we can convert some of this code back to being safe, and we need to know if we are regressing.